commons/src/main/res/layout/activity_manage_blocked_numbers.xml

```diff
@@ -1,43 +1,62 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/manage_blocked_numbers_wrapper"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="match_parent">
+
+    <com.simplemobiletools.commons.views.MySwitchCompat
+        android:id="@+id/block_unknown"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/activity_margin"
+        android:text="@string/block_unknown_calls"
```

Comment:
@tibbi because this is a shared screen between SMS and call apps we need another, generic title suitable for both apps.

Reply:
well, just use block_unknown_messages at SMS Messenger..

Reply:
This file is in the commons library.

Reply:
that doesnt prevent you from changing the text. Check config.appId to see the package name.

Reply:
You suggest checking the current app using the package name inside the commons library to set the correct text. 
Did I get it right?

Reply:
sure, but use the config.appId for it, casual context.packagename will return Simple Commons's package name. Just check some usages od appId

Reply:
I added another string that is just a reference to the needed text for the app.

Reply:
what? You dont need to create any new hacky string... I created both of them already.

Reply:
Just change the view text programmatically